webpack/components/OpenscapRemediationWizard/steps/ReviewHosts.js

```diff
@@ -161,8 +163,15 @@ const ReviewHosts = () => {
   const columns = {
     name: {
       title: __('Name'),
-      wrapper: ({ id, display_name: displayName }) => (
-        <a href={foremanUrl(`hosts/${id}`)}>{displayName}</a>
+      wrapper: ({ name, display_name: displayName }) => (
+        <Button
+          component="a"
+          variant="link"
+          target="_blank"
+          href={foremanUrl(`${useForemanHostDetailsPageUrl()}${name}`)} // eslint-disable-line react-hooks/rules-of-hooks
```

Comment:
Please move `useForemanHostDetailsPageUrl` out so it wont break react by accident

Reply:
It's really... weird that this could break anything :/

Updated.